cloudlift/deployment/service_template_generator.py

```diff
@@ -280,7 +284,20 @@ def _add_service(self, service_name, config):
                         Principal=Principal("Service", ["ecs-tasks.amazonaws.com"])
                     )
                 ]
-            )
+            ),
+            Policies=[
+                Policy(
+                    PolicyName=service_name + "-Firelens-" + hashlib.sha1(service_name.encode()).hexdigest()[0:5],
```

Comment:
Review the need for hashing

Reply:
Removed hashing, turned out to be redundant.